pkg/operator/helpers/helpers.go

```
@@ -1033,3 +1055,5 @@ func GRPCServerEndpointType(cm *operatorapiv1.ClusterManager) string {
 
 	return string(operatorapiv1.EndpointTypeHostname)
 }
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)